static/scss/partials/glocal-menu.scss

```
@@ -20,6 +20,14 @@ glocal-menu {
   right: 28px;
   border-radius: 8px;
   box-shadow: 0 7px 12px -3px rgba(28, 28, 29, .502);
+  width: auto;
+  max-width: $screen-sm;
+
+    @media screen and (max-width: 520px) {
```

This comment applies for both this `max-width` and the `max-width: 520px` lines. 

Could you adjust the `max-width` to be a `calc()` argument of `$screen-sm` minus the padding? 

Then you could leave the breakpoint below at a standard, variable number. 



```suggestion
  width: auto;
  max-width: calc($screen-sm - $spacing-xl);

    @media screen and (max-width: $screen-sm) {
```

---

You may need to double check if `calc()` can accept Sass variables like that. I recall it messing up with standard CSS variables, maybe?